functions/createMarkup.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Handle markup that contains HTML.
+ *
+ * @param {Array} props Array of JSX Objects.
```

Comment:
This doc block is missing both @author and @return comments. Furthermore, your code editor _should_ be catching these.

Reply:
VSCode isn't catching these. I'll have to investigate.